backend/utils.ts

```diff
@@ -342,6 +343,23 @@ export const stringWithAbbreviationsProcessor = (string: string): string => {
   return establishmentName
 }
 
+// Either explicitly enable redirect for enabled prisons or check for the developer role
+export const getRedirectEnabled = (res): boolean => {
+  const redirectUrl = config.app.prisonerProfileRedirect.url
+  const redirectDate = config.app.prisonerProfileRedirect.enabledDate
+  const redirectEnabled =
+    redirectUrl &&
+    ((redirectDate && redirectDate < Date.now()) ||
+      res.locals.user?.userRoles?.some((role) => role === 'ROLE_DPS_APPLICATION_DEVELOPER'))
+
+  return redirectEnabled
+}
+
+export const getRedirectCaseLoad = (activeCaseLoadId: string): boolean => {
```

Comment:
As above, probably better as isRedirectCaseLoad as it returns a boolean rather than a caseload